contrib/completion/fish/docker.fish

```diff
@@ -164,6 +168,42 @@ complete -c docker -A -f -n '__fish_seen_subcommand_from commit' -s m -l message
 complete -c docker -A -f -n '__fish_seen_subcommand_from commit' -s p -l pause -d 'Pause container during commit'
 complete -c docker -A -f -n '__fish_seen_subcommand_from commit' -a '(__fish_print_docker_containers all)' -d "Container"
 
+# config
+complete -c docker -f -n '__fish_docker_no_subcommand' -a config -d "Manage Docker configs"
+complete -c docker -f -n '__fish_seen_subcommand_from config' -a create -d 'Create a config from a file or STDIN'
+complete -c docker -f -n '__fish_seen_subcommand_from config' -a inspect -d 'Display detailed information on one or more configs'
+complete -c docker -f -n '__fish_seen_subcommand_from config' -a ls -d 'List configs'
+complete -c docker -f -n '__fish_seen_subcommand_from config' -a rm -d 'Remove one or more configs'
+
+# container
+complete -c docker -f -n '__fish_docker_no_subcommand' -a container -d 'Manage containers'
+complete -c docker -f -n '__fish_seen_subcommand_from container' -a attach -d 'Attach local standard input, output, and error streams to a running container'
+complete -c docker -f -n '__fish_seen_subcommand_from container' -a commit -d 'Create a new image from a container\'s changes'
+complete -c docker -f -n '__fish_seen_subcommand_from container' -a cp -d 'Copy files/folders between a container and the local filesystem'
+complete -c docker -f -n '__fish_seen_subcommand_from container' -a create -d 'Create a new container'
+complete -c docker -f -n '__fish_seen_subcommand_from container' -a diff -d 'Inspect changes to files or directories on a container\'s filesystem'
+complete -c docker -f -n '__fish_seen_subcommand_from container' -a exec -d 'Run a command in a running container'
+complete -c docker -f -n '__fish_seen_subcommand_from container' -a export -d 'Export a container\'s filesystem as a tar archive'
+complete -c docker -f -n '__fish_seen_subcommand_from container' -a inspect -d 'Display detailed information on one or more containers'
+complete -c docker -f -n '__fish_seen_subcommand_from container' -a kill -d 'Kill one or more running containers'
+complete -c docker -f -n '__fish_seen_subcommand_from container' -a logs -d 'Fetch the logs of a container'
+complete -c docker -f -n '__fish_seen_subcommand_from container' -a ls -d 'List containers'
```

Comment:
we should also have the aliases (`docker container ls`, `docker container list`); perhaps can be combined on a single line